### PR TITLE
Updated prettier.

### PR DESCRIPTION
**Pull request by mhaertwig**
- Newer version.
- Displaying errors using eslint-plugin-prettier

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -7,11 +7,13 @@
...
   "parser": "babel-eslint",
   "extends": ["airbnb", "prettier"],
+  "plugins": ["prettier"],
   "rules": {
     "import/no-extraneous-dependencies": ["error", {"devDependencies": true }],
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "react/forbid-prop-types": "Off",
     "max-len": ["error", { "code": 80, "ignoreComments": true }],
-    "react/jsx-curly-spacing": ["error", { "when": "never" }]
+    "react/jsx-curly-spacing": ["error", { "when": "never" }],
```
Comment (oterral) [inline .eslintrc.json]: you can remove max-len and react/jsx-curly-spacing we don't need it anymore, prettier takes care of it

**Comment by oterral** [comment]
@mhaertwig what do you think remove the `.prettierrc.json`  file if if we adopt it, we might as well adopt it fully.

**Comment by mhaertwig** [comment]
@oterral Nooo. The semicolon thing would be too much for me.

**Comment by mhaertwig** [comment]
The code would be a mess without prettier! I'm lucky that we have it.

**Comment by oterral** [comment]
You just join a big and happy family !!!

**Pull request merged**